pkg/eraser/eraser.go

```diff
@@ -189,15 +197,41 @@ func removeVulnerableImages(c Client, socketPath string, imagelistName string) (
 		}
 	}
 
-	// TODO: change this to read vulnerable images from ImageList
-	// adding random image for testing purposes
+	// get vulnerable images from ImageList
+	config, err := rest.InClusterConfig()
```

Comment:
This will mess up unit tests.
We could pass clientset into `removeVulnerableImages`, but I'm more inclined to pass it the list of images instead.